app/controllers/api/v1/admin/site_settings_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Admin
+      class SiteSettingsController < ApiController
+        def index
+          site_settings = Setting.joins(:site_settings)
+                                 .where(site_settings: { provider: 'greenlight' })
+                                 .pluck(:name, :value)
+                                 .to_h
+
+          return render_error status: :internal_server_error if site_settings.blank?
```

Comment:
Uhhhh why would this return an error?

Reply:
Yup, that was mine xD
What I thought about is what to do when having an  invalid `:provider` that causes the return of an empty data `{}` and I wanted to obscure the error.

But after giving it  a second thought this shouldn't happen, so removing that should be fine.